tests/test_newdep.py

```diff
@@ -10,6 +10,9 @@
 def test_omero_version(Command, Sudo):
     with Sudo('data-importer'):
         ver = Command.check_output("%s version" % OMERO)
+    # TODO: This will have to be updated with each major release
+    # These happen infrequently so hard code the version to reduce the
+    # chance of errors being missed elsewhere
     assert re.match('5\.3\.\d+-ice36-', ver)
```

Comment:
`\d+` - patch can only ever be a single digit, right? If so, perhaps drop the `+`?

Reply:
Not necessarily. We hit `4.4.11`

Reply:
You're right! I'd forgotten about that. 👍